modules/installation-configuration-parameters.adoc

```diff
@@ -1233,12 +1233,24 @@ Additional {rh-virtualization} configuration parameters are described in the fol
 |Required. The vNIC profile ID of the VM network interfaces. This can be inferred if the cluster network has a single profile.
 |String. For example: `3fa86930-0be5-4052-b667-b79f0a729692`
 
-|`platform.ovirt.api_vip`
+|`platform.ovirt.api_vips`
 |Required. An IP address on the machine network that will be assigned to the API virtual IP (VIP). You can access the OpenShift API at this endpoint.
```

Comment:
Actually it is possible to specify up to two IP addresses (in dual-stack case). Same applies for the other apiVip**s** and ingressVip**s** fields.

Reply:
Thanks @creydr - I have implemented your feedback. Please let me know if this is accurate or if any other issues. Thanks!